.idea/compiler.xml

```diff
@@ -22,5 +18,9 @@
         <processorPath useClasspath="true" />
       </profile>
     </annotationProcessing>
+    <bytecodeTargetLevel>
+      <module name="flutter-gui-tests.main" target="11" />
```

Comment:
Can we target java 11 if we still support 2018.3? I'm not sure what jvm it runs on.